### DISCUSSION TITLE
code-server and sftp

### DISCUSSION THREAD

**Discussion by mozhack** [Natizyskunk/vscode-sftp#297] [General]
Hi,

I have a code server, installed the sftp and I use a mount point to share files between code server, computers, etc.

When I open a file in SFTP extension, it works, but when I click Reveal in Local it saves an empty file.

another strange thing is that the DownloadOnOpen don’t work too.

someone with same problems that can share the conf with me?

thanks

**Comment by Natizyskunk** [comment]
@mozhack Could you please open an issue respecting [this](https://github.com/Natizyskunk/vscode-sftp/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=) template ?
